.github/workflows/ghcr_build.yml

```diff
@@ -0,0 +1,57 @@
+name: Build GHCR images
+
+on:
+  push:
+    branches: 
+      - "master"
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build:
```

Comment:
For maintenance reasons, I would extract the buildx to a separate action since we now use it in different places.

Reply:
I tried to refactor things. Let me know what you think about.

And I believe the Docker Hub user has changed since my initial commit.